src/eShop.AppHost/Program.cs

```diff
@@ -18,9 +18,14 @@
 
 var launchProfileName = ShouldUseHttpForEndpoints() ? "http" : "https";
 
+// To configure Azure Monitor Application Insights, add the following:
+// var insights = builder.AddConnectionString("myInsightsResource", "APPLICATIONINSIGHTS_CONNECTION_STRING");
```

Comment:
This can be replaced with a conditional such as 

``` c#
var appInsights = (builder.Configuration.GetConnectionString("myAppInsightsResource") is not null)? 
    builder.AddConnectionString(
        "myAppInsightsResource",
        "APPLICATIONINSIGHTS_CONNECTION_STRING")
    : null;
```